package.json

```diff
@@ -20,7 +20,7 @@
     },
     "dependencies": {
         "assert-plus": "0.1.5",
-        "bunyan": "0.23.1",
+        "bunyan": "1.8.10",
```

Comment:
any reason to not use 1.8.12? Guessing because you saw 1.8.10 looked like the latest 1.8.x from the master branch of node-bunyan/CHANGES.md.

https://github.com/trentm/node-bunyan/blob/1.x/CHANGES.md

Reply:
I don't recall a specific reason, although if you are telling me that there was something that made 1.8.10 look like the latest reason up until very recently that sounds plausible.  Retested with 1.8.12

```
npm test | ./node_modules/.bin/bunyan

> zkplus@0.4.0 test /home/jcsb/src/j/misc/node-zkplus
> istanbul test test/test.js | ./node_modules/.bin/faucet

✓ constructor tests
✓ setup
✓ connect timeout
✓ mkdirp
✓ creat no options
✓ create: ephemeral
✓ create: sequential
✓ create: ephemeral_sequential
✓ put not exists
✓ put overwrite
✓ get
✓ readdir
✓ update
✓ stat
✓ unlink
✓ getState
✓ toString
✓ teardown
✓ error when closed
✓ setup
✓ heartbeat
✓ stop proxy
✓ start proxy
✓ rewatch
✓ re-ephemeral
✓ teardown
✓ setup
✓ watch (data)
✓ watch (directory)
✓ watch (data+initialData)
✓ watch (directory + initial data)
✓ teardown
# tests 114
# pass  114
✓ ok

```